src/WebJobs.Extensions.DurableTask.CodeGeneration/WebJobs.Extensions.DurableTask.CodeGen.Example/WebJobs.Extensions.DurableTask.Codegen.Example.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)Generated</CompilerGeneratedFilesOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.8.5" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.5.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.4.3" />
```

Comment:
The project depends on the public version of ```Microsoft.Azure.WebJobs.Extensions.DurableTask```. This is not ideal, because it doesn't allow you to run the project to verify the code that will resolve the Generated dependencies works.

Reply:
Yea ```Microsoft.Azure.WebJobs.Extensions.DurableTask``` should be publishable standalone. It should be a no-op if no generated code is available. We could even publish that before having anything available for the SourceGenerator.